src/typeahead/index.js

```diff
@@ -66,6 +67,7 @@ var Typeahead = React.createClass({
       placeholder: "",
       textarea: false,
       inputProps: {},
+      preventKeyEvents: true,
```

Comment:
This seems to be named awkwardly.  The name suggests that setting it to **true** will prevent key events.  Perhaps this should be renamed to **propagateKeyEvents** and default to **false**?  Then the code in _onKeyDown would be
```
  if (!this.props.propagateKeyEvents) {
    event.preventDefault();
  }
```